finalfrontier-utils/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "finalfrontier-utils"
 version = "0.4.0"
+edition = "2018"
 authors = ["Daniël de Kok <me@danieldk.eu>"]
 description = "Train and use word embeddings with subword representations"
 documentation = "https://git.danieldk.eu/finalfrontier/about"
```

Comment:
Unrelated to this specific PR, but the documentation webpage results in Error 404

Reply:
Ah, yes, some metadata in some of the crates is stale. We should probably start using `docs.rs` for the `documentation` field and `website` to `finalfusion.github.io`.